libs/zzz/formula-ui/src/char/CharacterEditor.tsx

```diff
@@ -86,7 +153,7 @@ export function Content({ onClose }: { onClose?: () => void }) {
                   }
                 />
               </Box>
-              <CharacterCardStats bgt="light" character={character} />
+              <CharStatsDisplay />
```

Comment:
Stats would be empty in character page (Without a calc provider) 
![image](https://github.com/user-attachments/assets/a8f98e5c-8306-476d-8dcd-0b77cafc48f3)